src/Sample.xoo

```diff
@@ -10,3 +10,4 @@ public class Sample1 {
     System.out.println("hello" + " xoo");
   }
 }
+// xoo
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:44659/organizations/org2/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:44659/project/issues?id=sample&pullRequest=5373&issues=AWcL4AS8gYp4NVEm_H7y&open=AWcL4AS8gYp4NVEm_H7y)

[ISSUE_KEY]: AWcL4AS8gYp4NVEm_H7y